src/web/components/AuditTrail/AuditTrail.scss

```diff
@@ -0,0 +1,73 @@
+.audit-trail-table-container {
+  .audit-log-table {
+    font-size: 0.875rem;
+    width: 100%;
+
+    thead {
+      font-weight: 700;
+      border: none;
+    }
+
+    td {
+      padding-top: 15px;
+      padding-bottom: 15px;
+
+      &.succeeded {
+        width: 5%;
+        text-align: center;
+      }
+
+      &.event-data {
+        width: 50%;
+      }
+
+      .icon_x {
+        color: red;
+      }
+
+      .icon_check {
+        color: green;
+      }
+    }
+  }
+
+  .audit-trail-table-header {
+    display: flex;
+    justify-content: end;
+    align-items: baseline;
+    padding-bottom: 6px;
+
+    &-right {
+      display: flex;
+      justify-content: right;
+    }
+
+    .audit-trail-search-bar-container {
+      display: flex;
+      align-items: center;
+      border-bottom: 2px solid var(--theme-action);
+      padding: 6px 2px;
+      ::placeholder {
+        color: var(--theme-search-text);
+        opacity: 1;
+      }
+      margin-right: 20px;
```

Comment:
We can probably remove this padding so that it lines up with the end of the table, like so:
![image](https://github.com/user-attachments/assets/924800bb-5b70-433f-90bc-98a7ac0cd8f3)